packages/core/src/types.ts

```diff
@@ -1834,9 +1838,17 @@ export interface ActorBehavior<
     state: TInternalState,
     actorCtx: ActorContext<TEvent, TSnapshot>
   ) => TInternalState;
+  /**
+   * Clones behavior that starts at specified state
+   */
+  at?: (state: TInternalState) => this;
+  /**
+   * @returns Persisted state as JSON
```

Comment:
We don't completely mandate the nature of the persisted state. We could mention this as a recommendation here but we shouldn't imply here what doesn't have to be always true.
```suggestion
   * @returns Persisted state
```